src/sentry/static/sentry/app/providers/withOrganization.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import SentryTypes from '../proptypes';
+
+export default function withOrganization(WrappedComponent) {
```

Comment:
I personally think we should drop these "context" mixins/HOCs. I don't like that they introduce magic prop functions to access properties like `access` and `features`. 

I think it's easier and more clear for child components to just specify `contextTypes` and opt-in where appropriate. If we want utility methods like `getAccess`, let's declare them on the `organization` object itself when it's initialized (e.g. wrap the object in a utility class/object).

Reply:
I guess one benefit is that we could move these values to stores, and not go through `contextTypes` at all ... which could avoid some of the janky context propagation problems that can occur.